src/test/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/resource/SyncAndMigrateResourceIntTest.kt

```diff
@@ -943,7 +953,7 @@ class SyncAndMigrateResourceIntTest : SqsIntegrationTestBase() {
       }
 
       @Test
-      fun `can migrate a location different deactivationReason Local Work`() {
+      fun `can migrate a location different deactivationReason Maintenance`() {
```

Comment:
This is cause NomisReason is LOCAL_WORK however DeactivatedReason is Maintenance so I refactor